src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/tab.cljs

```diff
@@ -139,10 +219,21 @@
        (endpoints/call-ajax-orch
          {:endpoint (endpoints/get-workspace (:workspace-id props))
           :on-done (fn [{:keys [success? get-parsed-response status-text]}]
-                     (swap! state assoc :server-response
+                     (do
+                       (swap! state assoc :server-response
+                         (if success?
+                           {:workspace (get-parsed-response)}
+                           {:workspace-error status-text}))
                        (if success?
-                         {:workspace (get-parsed-response)}
-                         {:workspace-error status-text})
+                         (let [response (:server-response @state)
+                               r1 (utils/rlog (str "the response is " response))
+                               attributes (get-in response [:workspace "workspace" "attributes" ])
+                               r2 (utils/rlog (str "attrs is " attributes))
+                               attrs-list (mapv (fn [[k v]] [k v]) attributes)]
+                           (utils/rlog (str "the response is " response))
+                           (utils/rlog (str "the attrs-list is "attrs-list))
```

Comment:
Re "Take out logging calls"  I had them in there for so long I didn't notice them anymore.